content/docs/reference/cookies.mdx

```diff
@@ -161,6 +161,12 @@ cookie:
 
 ## Cookie Secure {#cookie-secure}
 
+:::caution
+
+As of **v0.25**, we have deprecated the **Cookie Secure** setting. Pomerium will always set the `Secure` attribute on session cookies going forward.
```

Comment:
Sorry, I didn't mean to imply that you should delete the "and will remove it in **v0.26**" part. The phrasing here might be a little confusing as to the behavior in v0.25.

What do you think about this?
```suggestion
As of **v0.25** the **Cookie Secure** setting is deprecated, and it will be removed in **v0.26**. Once removed, Pomerium will always set the `Secure` attribute on session cookies.
```

Reply:
That's fine. This was a weird process for this setting, so I appreciate the help with the messaging.